config/config/sc-config.yaml

```
@@ -265,6 +265,8 @@ prometheus:
           memory: 100Mi
   ## the percentage of use the resources are set to hit in 3 days
   predictLinearLimit: 66
+  s3BucketPercentLimit: 80
+  s3BucketMaxSizeLimit: 1000
```

If I read the code correctly then `s3BucketMaxSizeLimit` should be set to the quota set by the cloud provider and `s3BucketPercentLimit` is on what percentage of the quota we want to alert on. If so I would like to make it a bit more clear that the size limit is the quota.
```suggestion
  s3BucketPercentLimit: 80
  s3BucketSizeQuotaGB: 1000
```

---

Correct!